draft-ietf-ppm-dap.md

```diff
@@ -1470,6 +1472,10 @@ The named parameters are:
 
 * `task_id`, the DAP task ID. If the Leader does not recognize the task ID, it
   MUST abort with error `unrecognizedTask`.
+* `extensions`, a list of protocol extensions ({{protocol-extensions}}). If the
+  Leader does not recognize one of the extensions, or if two extensions have the
+  same ExtensionType, it MUST abort the request with error
+  "unrecognizedMessage".
```

Comment:
Continuing from my remark above: I think we should at least recommend against the leader ever parsing extensions. HTTP already provides plenty of mechanisms for the collector to provide extra information or parameters to the leader out-of-band from DAP protocol messages. Extensions should only ever be needed so the collector can say something extra to the helper, and at that point we have to think about how the leader could forge extensions. Maybe there should be a recommendation that anything put into an extension by the collector needs to be authenticated so the helper can verify where it came from?